project/Newevaluation.html

```diff
@@ -110,253 +143,221 @@
     }
 
     caption{
-      margin-left: -800px;
-      font-size: 25px;
+      margin-left: -84%;
+      font-size: 1.6rem;
       color: #03A9F4;
-      margin-bottom: 10px;
+      margin-bottom: 0.6rem;
     }
 
 
+/* ----------------------- Should the candidate be hired? ------------------- */
+
     .secondBox {
       background-color: whitesmoke;
-      margin-top: -21px;
-      width: 1895px;
-      height: 200px;
+      margin-top: 2rem;
+      width: 100%;
+      height: 12rem;
     }
 
     h2 {
-      margin-left: 410px;
+      margin-left: 25.5rem;
       background-color: whitesmoke;
-      font-size: 25px;
+      font-size: 1.6rem;
       color: #03A9F4;
-      margin-bottom: -30px;
+      margin-bottom: -2.5rem;
       font-weight: normal;
     }
 
 
     textarea {
-      border: 1px solid black;
-      width: 1150px;
-      height: 80px;
-      margin-left: 400px;
-      margin-top: 50px;
-      font-style:italic;
+      border: 1px solid grey;
+      width: 72rem;
+      height: 6.5rem;
+      margin-left: 21%;
+      margin-top: 2.5%;
+      font-style: italic;
       background-color: whitesmoke;
-      font-size: 15px;
+      font-size: 1.1rem;
       color: grey;
-    }
+      }
+
+
+/* -------------------------------General Impression------------------------- */
 
     .thirdBox {
       background-color: whitesmoke;
-      margin-top: -21px;
-      width: 1895px;
-      height: 200px;
+      margin-top: 2rem;
+      width: 100%;
+      height: 12rem;
     }
 
+
+
+/* ------------------------Workflow, Leadership & Soft skills-----------------*/
+
     .fourthBox {
       background-color: whitesmoke;
-      margin-top: -21px;
-      width: 1895px;
-      height: 200px;
+      margin-top: 2rem;
+      width: 100%;
+      height: 12rem;
     }
 
+
+/* -------------------------------OOP, Design Patterns------------------------*/
+
     .fifthBox {
       background-color: whitesmoke;
-      margin-top: -21px;
-      width: 1895px;
-      height: 250px;
-      padding-top: 30px;
+      margin-top: 2rem;
+      width: 100%;
+      height: 16rem;
+      padding-top: 0rem;
```

Comment:
Could we write this as `0px` instead? What about `0%`? `0vw`? Would it make any difference? :)